lib/location/method_cellular.c

```diff
@@ -0,0 +1,209 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <stdio.h>
+#include <zephyr.h>
+#include <logging/log.h>
+
+#include <modem/modem_info.h>
+#include <modem/location.h>
+#include <modem/lte_lc.h>
+
+#include <net/multicell_location.h>
+
+#include "loc_core.h"
+#include "loc_utils.h"
+
+LOG_MODULE_DECLARE(location, CONFIG_LOCATION_LOG_LEVEL);
+
+BUILD_ASSERT(
+	(int)LOC_SERVICE_ANY == (int)MULTICELL_SERVICE_ANY &&
+	(int)LOC_SERVICE_NRF_CLOUD == (int)MULTICELL_SERVICE_NRF_CLOUD &&
+	(int)LOC_SERVICE_HERE == (int)MULTICELL_SERVICE_HERE &&
+	(int)LOC_SERVICE_SKYHOOK == (int)MULTICELL_SERVICE_SKYHOOK,
+	"Incompatible enums loc_service and multicell_service");
```

Comment:
I don't see this breaking unless someone tries to do it intentionally. On the other hand, I don't like mapping functions which assign 1 = 1 values, and when I saw this used somewhere I thought it would do the trick.
We can discuss this tomorrow unless I give up and remove it today :)

Reply:
It adds an unnecessary dependency to the Multicell lib, but most likely it will never cause any problems. I don't see why the IDs would be changed. I'm not saying this needs to be changed, does anybody else have any opinion?

Reply:
This was accepted based on discussions.